### PR TITLE
break: Remove luxon

### PR DESCRIPTION
**Pull request by nickredmark**

### CODE REVIEW

File: src/resolvers/mutations.ts [hunk 1]
```diff
@@ -365,7 +364,7 @@ const sanitize = (ctx: FullContext, model: EntityModel, data: Entity) => {
     }
 
     if (isEndOfDay(field) && data[key]) {
```
Comment (dwirz) [inline src/resolvers/mutations.ts:366]:
```suggestion
    if (isEndOfDay(field) && typeof data[key] === 'string') {
```
so you do not need ` as string` below, wdyt?

Comment (nickredmark) [reply src/resolvers/mutations.ts:366]: we discussed

File: src/bin/gqm/settings.ts [hunk 2]
```diff
@@ -52,6 +61,32 @@ const DEFAULTS = {
...
+    init: async (dateLibrary: DateLibrary) => {
+      const knexfilePath = await getSetting('knexfilePath');
+      switch (dateLibrary) {
+        case 'luxon': {
+          const timeZone = await getSetting('timeZone');
+          ensureFileContains(knexfilePath, 'luxon', KNEXFILE_LUXON_TYPE_PARSERS(timeZone));
+          break;
+        }
+        case 'dayjs':
+          ensureFileContains(knexfilePath, 'luxon', KNEXFILE_DAYJS_TYPE_PARSERS);
```
Comment (dwirz) [inline src/bin/gqm/settings.ts:76]:
```suggestion
          ensureFileContains(knexfilePath, 'dayjs', KNEXFILE_DAYJS_TYPE_PARSERS);
```

File: docs/docs/11-luxon.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Luxon
+
+This is a recipe
+
+```ts
+import { DateTime } from 'luxon';
+
+
+
```
Comment (dwirz) [inline docs/docs/11-luxon.md:9]:
```suggestion

```

Comment (nickredmark) [reply docs/docs/11-luxon.md]: removed this file given that gqm takes care of it

**Pull request merged**